src/send.js

```diff
@@ -32,8 +35,14 @@ let sendDataPoints = async (configurer, options, payload) => {
       },
       encoding: null,
       body:     serialize(payload),
-      timeout:  30 * 1000,
-      retries:  5,
+      timeout:  90 * 1000,
+      retries:  (iter, error) => {
```

Comment:
But yeah is 10min too much? I'm thinking it might be okay... After all, we only send every what 30 - 90 s or so?